src/sidebar/templates/annotation.html

```diff
@@ -37,14 +37,14 @@
       h-branding="accentColor">mire</a>
   </div>
   <!-- Tags -->
-  <div class="annotation-body form-field" ng-if="vm.editing()">
+  <div class="annotation-body" ng-if="vm.editing()">
```

Comment:
`.form-field` had extant rules, but they did nothing here as all of them were overridden by more specific rules.